project/Build.scala

```diff
@@ -1135,19 +1135,41 @@ object Build {
                 IO.copyFile(referenceStdlibPaths, destination)
               }
             }
+          case "diff" +: rest =>
+            log.info(s"Diffing ${name.value}/src with scala-library sources")
+            if (rest.size > 1) {
+              log.error(s"Too many arguments for $projectName/run diff")
+            } else {
+              val path = rest.headOption.getOrElse("")
+              val fullPath = srcDir / path
+              if (!fullPath.exists) {
+                log.error(s"$fullPath does not exist")
+              } else {
+                val command = s"git diff --diff-filter=ACMR --no-index --color=always -- $reference/$path $fullPath"
```

Comment:
`--diff-filter=ACMR` is missing `D` on purpose not to show all the files that have not been overwritten.

Reply:
Put this in code comment instead?

Reply:
Indeed. Added.